3.0/vector_tile.proto

```diff
@@ -92,14 +92,28 @@ message Tile {
                 // The first increment after a null is given relative
                 // to the last non-null value, or to global_offset_*
                 // if there was no previous value.
+                //
+                // There is one `extended_geometry` element per
+                // DimensionalMapping per `moveto` or `lineto` in the
+                // `geometry`. If there are multiple DimensionalMappings,
+                // all the dimensions for each `moveto` or `lineto` are
+                // given in sequence before moving on to the dimensions
+                // for the next `moveto` or `lineto`.
```

Comment:
Do we need one for `closepath` also? It introduces an additional point implicitly which has the same x/y coordinates as the first point in the path but might have different values in this extra dimension (for instance if the dimension is the time).

Reply:
I could see this argument for the non-geometric `node_attributes`, but I am taking the position that anything in the geometry has to have a spatial interpretation: in other words, that it is OK to interpolate between nodes, and that `closepath`'s claim of spatial continuity applies in all dimensions. I don't think a `closepath` that only closes the path in some dimensions makes sense.